subsystem/wrist.py

```diff
@@ -36,6 +36,8 @@ def __init__(self):
         self.wrist_moving: bool = False
         self.coral_in_feed: bool = False
 
+        self.detected_time = 0
```

Comment:
I think that we can avoid this weird thing about making the detected time zero by using some methods in the Timer class.